README.md

```diff
@@ -292,3 +293,4 @@ Questions can be asked by raising an `Issue`.
 - [Vladyslav Kyiashko](https://github.com/vlakyi)
 - [Yashasvi Misra](https://github.com/yashasvimisra2798)
 - [Yaswanth Modepalli](https://github.com/Yaswanth14)
+
```

Comment:
@Mayurmg Can you remove the last line of the file.

Reply:
@adityaraute I guess that's because of linter. And removing it may cause linting errors

Reply:
Hasn't happened with other cases so perhaps not.